tooling/debugger/src/source_code_printer.rs

```diff
@@ -266,12 +267,20 @@ mod tests {
 
         // We don't care about opcodes in this context,
         // we just use a dummy to construct debug_symbols
-        let mut opcode_locations = BTreeMap::<OpcodeLocation, Vec<Location>>::new();
-        opcode_locations.insert(OpcodeLocation::Acir(42), vec![loc]);
+        let mut opcode_locations = BTreeMap::<AcirOpcodeLocation, CallStackId>::new();
+        opcode_locations.insert(AcirOpcodeLocation::new(42), CallStackId::new(1));
+        let mut location_tree = LocationTree::default();
+        location_tree
+            .locations
+            .push(LocationNodeDebugInfo { parent: None, value: Location::dummy() });
```

Comment:
Just for my understanding why do we push a `None` and dummy location here first instead of just starting with the root() and loc?

Reply:
The (None, dummy location) is the root, so that's what we do. Root is index 0, we need to have something at index 0 to represent it. We need a root to account for empty call stack.